LSpec/LSpec.lean

```diff
@@ -19,6 +19,7 @@ testing frameworks
 
 namespace LSpec
 
+open Lean
```

Comment:
Is this necessary? I try to avoid opening namespaces like that. This file compiles just fine without this line